tests/networkflows_test.go

```diff
@@ -64,7 +70,7 @@ func TestStackroxNetworkFlows(t *testing.T) {
 	for {
 		select {
 		case <-timeout.C:
-			t.Fatal("Failed to get the correct edges in 5 minutes")
+			s.T().Fatal("Failed to get the correct edges in 5 minutes")
```

Comment:
[nit] I'm not sure if `s.FailNow` would be better here.

Reply:
I think we're *technically* already using `FailNow`, checking the docs string on `Fatal`:
`// Fatal is equivalent to Log followed by FailNow.`
So I think we can leave it like this :smile: